src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -107,7 +107,7 @@ private function includeTaxes()
      *
      * @return ProductLazyArray|ProductListingLazyArray
      */
-    private function presentProduct(array $rawProduct)
+    public function presentProduct(array $rawProduct)
```

Comment:
You can call `Cache::clean('presentedCart_*'` this will clear the presenter cache, the complex part now is to find the appropriate place/moment when this should be cleared

Reply:
I took your feedback into account :+1: